src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs

```diff
@@ -72,7 +72,13 @@ public APIRevisionsManager(
         /// <returns></returns>
         public async Task<PagedList<APIRevisionListItemModel>> GetAPIRevisionsAsync(ClaimsPrincipal user, PageParams pageParams, APIRevisionsFilterAndSortParams filterAndSortParams)
         {
-             return await _apiRevisionsRepository.GetAPIRevisionsAsync(user, pageParams, filterAndSortParams);
+            var revisions = await _apiRevisionsRepository.GetAPIRevisionsAsync(user, pageParams, filterAndSortParams);
+            List<APIRevisionListItemModel> upgradedList = [];
+            foreach (var item in revisions)
+            {
+                upgradedList.Add(await UpgradeAPIRevisionIfRequired(item));
```

Comment:
What do you plan to do with the updatedList here?

Reply:
My bad. I forgot to update it when I reverted one change. Fixing it.